cmd/influxd/run/server.go

```diff
@@ -135,6 +135,9 @@ func NewServer(c *Config, buildInfo *BuildInfo) (*Server, error) {
 	s.PointsWriter.ShardWriter = s.ShardWriter
 	s.PointsWriter.HintedHandoff = s.HintedHandoff
 
+	// needed for executing into queries.
```

Comment:
Minor:  can we capitalize `INTO`? That gives the next a guy a clue that it's for that keyword (since the convention is we often capitalize keywords in docs).